udata/tests/test_issues.py

```diff
@@ -178,6 +178,36 @@ def test_list_issues_closed_filter(self):
         for issue in response.json['data']:
             self.assertIsNone(issue['closed'])
 
+    def test_list_issues_for(self):
+        dataset = Dataset.objects.create(title='Test dataset')
+        issues = []
+        for i in range(3):
+            user = UserFactory()
+            message = Message(content=faker.sentence(), posted_by=user)
+            issue = Issue.objects.create(
+                subject=dataset,
+                user=user,
+                title='test issue {}'.format(i),
+                discussion=[message]
+            )
+            issues.append(issue)
+        # Creating a closed issue that shouldn't show up in response.
+        user = UserFactory()
+        other_dataset = Dataset.objects.create(title='Other Test dataset')
+        message = Message(content=faker.sentence(), posted_by=user)
+        issue = Issue.objects.create(
+            subject=other_dataset,
+            user=user,
+            title='test issue {}'.format(i),
+            discussion=[message]
```

Comment:
`discussion` property is a legacy naming which might need renaming (in another PR) because now we have a `Discussion` model.